src/app/treasure-hunt/convert-input-data.service.ts

```diff
@@ -332,6 +337,61 @@ export class ConvertInputDataService {
     return airLeakSurveys;
   }
 
+  convertFlueGasLosses(flueGasLosses: Array<FlueGasTreasureHunt>, oldSettings: Settings): Array<FlueGasTreasureHunt> {
+    flueGasLosses.forEach(flueGas => {
+      if (flueGas.baseline.flueGasType == 'By Volume') {
+        flueGas.baseline.flueGasByVolume = this.convertFlueGasInput(flueGas.baseline.flueGasByVolume, oldSettings);
+        flueGas.modification.flueGasByVolume = this.convertFlueGasInput(flueGas.modification.flueGasByVolume, oldSettings);
```

Comment:
I think this is a problem in the other areas but it's not guranteed a modification exists. Should add a check for all